src/javascript/app_2/App/Components/Form/ButtonToggleMenu/button-highlight.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React     from 'react';
+
+const Highlight = ({ left, width }) => {
+    const highlight_style = {
+        width,
+        transform                : `translate3d(${left}px, 0, 0)`,
+        'borderTopLeftRadius'    : (left === 0) ? '4px' : '0',
```

Comment:
Can we add a `const border_radius_size = '4px'` and use it here and below instead of `4px`?